teerex/src/migrations.rs

```diff
@@ -0,0 +1,150 @@
+use super::*;
+
+use frame_support::{pallet_prelude::*, storage_alias, traits::OnRuntimeUpgrade};
+
+/// The log target.
+const TARGET: &str = "teerex::migration::v1";
+
+mod v0 {
+	use super::*;
+
+	#[derive(
+		Encode, Decode, Default, Copy, Clone, PartialEq, Eq, sp_core::RuntimeDebug, TypeInfo,
```

Comment:
I don't think it really matters, but I guess we don't need any of these derives for the old types? However, deriving from comments of parity engineers I read, I think that the rust compiler doesn't even include unneeded implementations of types any more - so it doesn't hurt at all.

Reply:
true.